tests/plugins/identity/user.py

```diff
@@ -0,0 +1,71 @@
+from datetime import datetime
+from typing import Callable, Protocol, TypeAlias, TypedDict, Unpack, final
+
+import pytest
+from mimesis.locales import Locale
+from mimesis.schema import Field, Schema
+
+from server.apps.identity.models import User
+
+
+class UserData(TypedDict, total=False):
+    email: str
+    first_name: str
+    last_name: str
+    date_of_birth: datetime
+    address: str
+    job_title: str
+    phone: str
+    phone_type: int
+
+
+UserAssertion: TypeAlias = Callable[[str, UserData], None]
+
+
+@final
+class RegistrationData(UserData, total=False):
```

Comment:
suggestion: Remove `total=false`

By default, all attributes are required. Isn't this the case with `password1` and `password2`?
```suggestion
class RegistrationData(UserData):
```